openadmet/models/eval/binary.py

```diff
@@ -0,0 +1,101 @@
+from sklearn.metrics import precision_score, recall_score, confusion_matrix, ConfusionMatrixDisplay
+import matplotlib.pyplot as plt
+import numpy as np
+import wandb
+import pandas as pd
+
+from openadmet.models.eval.eval_base import EvalBase, evaluators
+
+
+@evaluators.register("PosthocBinaryMetrics")
+class PosthocBinaryMetrics(EvalBase):
+
+    """
+    Intended to be used for regression-based models to calculate
+    precision and recall metrics for user-input cutoffs
+
+    Not intended for binary models
+    """
+
+    def evaluate(self, y_true:list = None, y_pred:list = None, cutoff:float = None, report:bool = False, output_dir:str = None):
+        """
+        Evaluate the precision and recall metrics for model with user-input cutoffs.
+
+        Parameters:
+        y_true (array-like, optional): True values.
+        y_pred (array-like, optional): Predicted values.
+        cutoffs (list, optional): List of cutoff values to calculate precision and recall.
+        report (bool, optional): Whether to save jsons of resulting precision/recall metrics. Default is False.
+        output_dir (str, optional): Directory to save the output plots and report. Default is None.
+
+        Returns:
+        None
+        """
+
+        if y_true is None or y_pred is None:
+            raise ValueError("Must provide y_true and y_pred")
+        self.plot_confusion_matrix(y_true, y_pred, cutoff, output_dir)
+        self.plot_posthoc_classification(y_true, y_pred, cutoff, output_dir)
+        precision, recall = self.get_precision_recall(y_pred, y_true, cutoff)
+        self.report(report, output_dir, precision=precision, recall=recall)
+
+    def get_precision_recall(self, y_pred:list, y_true:list, cutoff:float):
+        """
+        Calculate precision and recall metrics for given cutoffs.
+
+        Parameters:
```

Comment:
Nitpick:  Numpy style docstrings https://numpydoc.readthedocs.io/en/latest/format.html

```suggestion
        Parameters
       ------------
```